### DISCUSSION TITLE
PHP Version 8.1.18 issue

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2236] [General]
I have got an errors while running Cloudlog on php version 8.1.18 on the general log view screen:

```
A PHP Error was encountered
Severity: 8192
Message: strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated
Filename: partial/log_ajax.php
Line Number: 106

A PHP Error was encountered
Severity: 8192
Message: strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated
Filename: partial/log_ajax.php
Line Number: 115
```
![Снимок экрана от 2023-07-02 18-13-08](https://github.com/magicbug/Cloudlog/assets/16667173/1d850abd-397f-42f7-81e9-0e74ae929428)

**Comment by AndreasK79** [comment]
Which branch are you on? I think this is at least fixed in the dev branch.

**Comment by R1BLH** [comment]
> Which branch are you on? I think this is at least fixed in the dev branch.

Just updated to the latest dev branch. Really fixed! Thanks a lot!
